core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -384,19 +376,18 @@ extern "C" fn new_confirm_properties(n_args: usize, args: *const Obj, kwargs: *m
         let mut iter_buf = IterBuf::new();
         let iter = Iter::try_from_obj_with_buf(items, &mut iter_buf)?;
         for para in iter {
-            let [key, value, is_mono]: [Obj; 3] = iter_into_objs(para)?;
-            let key = key.try_into_option::<StrBuffer>()?;
-            let value = value.try_into_option::<StrBuffer>()?;
+            let [key, value]: [Obj; 2] = iter_into_objs(para)?;
```

Comment:
let's keep `is_mono`, it will come in handy when we want to indicate non-bytes properties that should be monospaced

Reply:
de9937d